codalab/codalabtools/compute/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+RUN mkdir -p /app
+COPY . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
+RUN wget -O codalabtools.py https://raw.githubusercontent.com/jared-cain/codalab-competitions/jared-dev/codalab/codalabtools/__init__.py && \
```

Comment:
Can we not hard code this? Should be able to grab relative to our directory, somehow?